assets/dev/js/editor/components/settings/base/manager.js

```diff
@@ -1,4 +1,5 @@
 var ControlsCSSParser = require( 'elementor-editor-utils/controls-css-parser' );
+import { Sources } from 'elementor-editor/editor-constants';
```

Comment:
Please remove the `activateTab` from component base in order to add the correct meta for the panel and template-library.